apps/admission-webhook/upstream/base/mutating-webhook-configuration.yaml

```diff
@@ -16,7 +16,7 @@ webhooks:
   name: $(podDefaultsDeploymentName).kubeflow.org
   namespaceSelector:
     matchLabels:
-      app.kubernetes.io/part-of: kubeflow-profile
+       opendatahub.io/dashboard: "true"
```

Comment:
I know the dashboard team is looking at moving away from making this label required on "data science project" namespaces.  This may cause issues in the future if they do move away from requiring this.

Reply:
Thanks for letting me know. Is there a better alternative for a consistent matching label to use here?

Reply:
They are mostly looking at moving away from labels on data science project namespaces because setting labels via self-provisioning of projects / ProjectRequests is impossible and namespace-access for setting the labels after project creation is really too much.

Reply:
Maybe we could achieve that with project templates. There would be the standard default template for std projects, and a special template for DSPs that would set labels and anything else needed. That would be the template called by dashboard when creating new DSP, with:
```
projectRequestTemplate:
    name: <template_name>
```
in the specs of the project.

Reply:
I in the past experienced trouble with anything but the standard few labels on a template: 

https://docs.openshift.com/container-platform/4.10/applications/projects/configuring-project-creation.html#about-project-creation_configuring-project-creation

Same for annotations. The argument at RedHat is: both annotations and labels should not be left to self-provisioners.

Reply:
FWIW, the discussion around making a UI that Trevor noted is the effort behind https://github.com/opendatahub-io/odh-dashboard/issues/1417